File: src/main/java/jp/artan/japanesefoodmod/init/tabs/RawMaterialsCreativeTab.java
package jp.artan.japanesefoodmod.init.tabs;

import net.minecraft.creativetab.CreativeTabs;
import net.minecraft.item.ItemStack;

import static jp.artan.japanesefoodmod.init.BlockInit.SOY;

public class RawMaterialsCreativeTab extends CreativeTabs {

    public RawMaterialsCreativeTab() {
        super("japanesefoodrawmaterials");
    }

    @Override
    public ItemStack getTabIconItem() {
        return new ItemStack(SOY.Seed);
    }

}

File: src/main/java/jp/artan/japanesefoodmod/init/tabs/RawMaterialsCreativeTab.kt
package jp.artan.japanesefoodmod.init.tabs

import jp.artan.japanesefoodmod.init.BlockInit
import net.minecraft.creativetab.CreativeTabs
import net.minecraft.item.ItemStack

class RawMaterialsCreativeTab : CreativeTabs("japanesefoodrawmaterials") {


    override fun getTabIconItem(): ItemStack {
        return  ItemStack(BlockInit.SOY.Seed)
    }

}
